src/overlay/src/Overlay.js

```diff
@@ -181,13 +181,29 @@ class Overlay extends React.Component {
         const autofocusElement = this.containerElement.querySelector(
           '[autofocus]'
         )
-        const wrapperElement = this.containerElement.querySelector('[tabindex]')
-        // eslint-disable-next-line eqeqeq, no-eq-null
+
+        // eslint-disable-next-line eqeqeq, no-eq-null, no-negated-condition
         if (autofocusElement != null) {
           autofocusElement.focus()
```

Comment:
You should move the `autofocusElement.focus()` to the else and vise versa instead of ignoring the problem.